sdk/storage/storage-blob/review/generated/src/models/index.d.ts

```diff
@@ -0,0 +1,6713 @@
+/// <reference types="node" />
+import { ServiceClientOptions } from "@azure/core-http";
+import * as coreHttp from "@azure/core-http";
+/**
+ * Key information
+ */
+export interface KeyInfo {
+    /**
+     * The date-time the key is active in ISO 8601 UTC time
+     */
+    start: string;
+    /**
+     * The date-time the key expires in ISO 8601 UTC time
+     */
+    expiry: string;
+}
+/**
+ * A user delegation key
+ */
+export interface UserDelegationKey {
+    /**
+     * The Azure Active Directory object ID in GUID format.
```

Comment:
might it be worth having a `type GUID = string` somewhere and then using `GUID` everywhere to make this clear without having to read the comments?

Reply:
I agree for myself, but people may assume they need to provide some object that's a guid.